### ISSUE TITLE
Command failed with exit code 65

### ISSUE THREAD

**Issue by joshigj15** [dpa99c/cordova-plugin-firebasex#421]
# Bug report

The following build commands failed:
	CompileC /Users/gorangjoshi/Library/Developer/Xcode/DerivedData/GANTAVYA-bnuemstkdmwjddaeyiajctoyuirq/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/AppAuth.build/Objects-normal/x86_64/OIDExternalUserAgentIOSCustomBrowser.o /Users/gorangjoshi/Desktop/Gantavya/mobile-app/platforms/ios/Pods/AppAuth/Source/AppAuth/iOS/OIDExternalUserAgentIOSCustomBrowser.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

Expected behavior:
It should compile properly.

Steps to reproduce:
ionic cordova build ios --prod

Environment information

Cordova CLI version
9.0.0 (cordova-lib@9.0.1)
Cordova platform version
Installed platforms:
  android 8.1.0
  browser 6.0.0
  ios 5.1.1

Plugins & versions installed in project (including this plugin)
"cordova-plugin-add-swift-support": "^2.0.2",
        "cordova-plugin-advanced-http": "^2.2.0",
        "cordova-plugin-androidx": "^1.0.2",
        "cordova-plugin-androidx-adapter": "^1.1.1",
        "cordova-plugin-app-version": "^0.1.9",
        "cordova-plugin-calendar": "^5.1.5",
        "cordova-plugin-camera": "^4.1.0",
        "cordova-plugin-crop": "^0.4.0",
        "cordova-plugin-datepicker": "^0.9.3",
        "cordova-plugin-file": "^6.0.2",
        "cordova-plugin-file-transfer": "^1.7.1",
        "cordova-plugin-filepath": "^1.5.7",
        "cordova-plugin-firebasex": "^9.1.1",
        "cordova-plugin-geolocation": "^4.0.2",
        "cordova-plugin-ionic-webview": "^4.1.3",
        "cordova-plugin-nativegeocoder": "^3.2.2",
        "cordova-plugin-nativestorage": "^2.3.2",
        "cordova-plugin-network-information": "^2.0.2",
        "cordova-plugin-sms-retriever-manager": "0.0.2",
        "core-js": "^2.5.4",

Dev machine OS and version, e.g.
mac OS 10.14.6

Node JS version
v12.16.3
XCode version
10.3.0

plugin version :- 
9.1.1

I am new to ionic please help me to get rid of this issue

**Comment by dpa99c** [comment]
https://github.com/dpa99c/cordova-plugin-firebasex#cordova-cli-builds

TL;DR: `9.1.1` => `9.1.1-cli`

**Comment by joshigj15** [comment]
update the plugin still get same issue 
please help me to get rid of this issue

**Comment by sagrawal31** [comment]
@joshigj15, as @dpa99c mentioned, ave you tried `9.1.1-cli` version?

**Comment by joshigj15** [comment]
yes sir I tried still having same issue??
